index.js

```diff
@@ -17,6 +17,38 @@ function Changelog( options ) {
 	this.options = options;
 	this.repo = new Repo( this.options.repo );
 
+	var ticketTypes;
+	if ( options.ticketTypes ) {
+		ticketTypes = options.ticketTypes;
+	} else {
+		ticketTypes = [ "github" ];
+	}
+
+	this.ticketTypes = [];
+	// Re-order ticketTypes according to the key order in Changelog.ticketParsers
```

Comment:
But why are you regenerating the list with a potentially different order? The option is what defines the order.